firstvoices/backend/tests/test_apis/test_category_api.py

```diff
@@ -211,45 +219,96 @@ def test_contains_multiple(self):
             str(category_both.id),
         ]
 
-        assert len(actual_ids) == len(ids_in_response)
-        # Checking all ids match up
-        difference = set(actual_ids) ^ set(ids_in_response)
-        assert not difference
+        assert set(actual_ids) == set(ids_in_response)
```

Comment:
Minor: could you please change "actual_ids" to "expected_ids" since it is the test expectation? It's convention to assert "expected" vs "actual" so it's confusing to use "actual" for the expectation